### DISCUSSION TITLE
Anschluss Vaillant geoTherm Plus

### DISCUSSION THREAD

**Discussion by wHyEt** [john30/ebusd#1034] [Q&A]
Hallo Zusammen,
hab jetzt meinen Ebus Adapter erhalten juhu und danke John:) 

Ich hätte jetzt noch eine frage ob jemand Erfahrung hat mit dem Anschluss an eine geoTherm Plus.
Habt ihr da den eBus über den service port abgegriffen (RJ11) oder habt ihr euch zwischen das display geschalten?

Was funktioniert besser? 

Vielen Danke für die Infos
Grüße
Nick

**Comment by ducktrA** [comment]
looking for something similar. controlling a vaillant arotherm using ebusd. Any insights?

**Reply by wHyEt** [reply]
I added it to the end of the Display it looks Like it works pretty well.

**Comment by ducktrA** [comment]
Thank you @wHyEt
I think you mean adding it as another device to the bus. Do you monitor the system or do you actively control it like switching it on or adjusting any temperature level?

Thx and best

**Reply by wHyEt** [reply]
at the moment i just monitor the system but I want to control it in the summer time wenn I have extra solar energy
